File: EGIS_Management/Domain/Entities/toims/FormIssuance.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities.toims
{
    public class FormIssuance
    {
        [Key] public int Id { get; set; }
        public string BookNumber { get; set; }
        public int Quantity { get; set; }
        public int StartReceipt { get; set; }
        public int EndReceipt { get; set; }
        public string Char { get; set; }
        [ForeignKey("Teller")] public int TellerCode { get; set; }
        public DateOnly FinalDate { get; set; }
        public DateTime DateAssigned { get; set; }
        public Teller Teller { get; set; }
    }
}

File: SPA19/EGIS/src/app/toims/models/formissuance.ts
export interface IFormIssuance {
    id: number;
    bookNumber : number;
    quantity : number;
    startReceipt : number;
    endReceipt : number;
    char: string
    tellerCode : number;
    tellerName : string;
    finalDate : Date;
    dateAssigned: Date;
 }

 export type createFormIssuanceDto =Omit<IFormIssuance,'bookNumber'|'quantity'|'startReceipt'|'endReceipt'|'char'|'tellerCode'|'tellerName'|'finalDate'|'dateAssigned'>; 